testing/stubtestclient/main.go

```diff
@@ -79,7 +79,10 @@ func genCode() string {
 	query.Set("medium", medium)
 	query.Set("source", source)
 	query.Set("experiment", experiment)
-	query.Set("installer_type", installerType)
+	// osx doesn't seem to support this attribution key
```

Comment:
do you know why by chance?

Reply:
It's not relevant here. On Windows, this is `full` or `stub` based on which NSIS installer is used. (Although...it looks like this might be out of date? I don't see this in https://searchfox.org/mozilla-central/rev/877adfeec838cdad97369441865bac605109427d/browser/components/attribution/AttributionCode.sys.mjs#47.)

Reply:
Yeah, I understand that this is not relevant but I am curious: if we start to have attribution params that are platform-dependent, and the validation of said params is different too, then it'd be good to document all that.



> Although...it looks like this might be out of date? I don't see this in [searchfox.org/mozilla-central/rev/877adfeec838cdad97369441865bac605109427d/browser/components/attribution/AttributionCode.sys.mjs#47](https://searchfox.org/mozilla-central/rev/877adfeec838cdad97369441865bac605109427d/browser/components/attribution/AttributionCode.sys.mjs#47)

bah, yeah? Maybe this is no longer relevant and Firefox doesn't use any `installer_type` anymore?

Reply:
Does this mean I can remove this exception for osx?

Reply:
I did some digging on this. It looks to me like there was [originally a thought that this would be needed](https://github.com/mozilla/bedrock/issues/8910), but based on what I see in https://github.com/mozilla/bedrock/issues/8963, Bedrock never added it to the attribution data.

This should be entirely safe to remove.